docs/source-pytorch/advanced/model_parallel/fsdp.rst

```diff
@@ -160,7 +160,7 @@ We can specify a list of layer classes in the **wrapping policy** to inform FSDP
         # 3. Pass it to the FSDPStrategy object
         strategy = FSDPStrategy(auto_wrap_policy=policy)
 
-    PyTorch provides several of these functional policies under :mod:`torch.distributed.fsdp.wrap`.
```

Comment:
This was correct


See https://github.com/pytorch/pytorch/blob/main/torch/distributed/fsdp/wrap.py

Reply:
maybe by source code but not according generated docs: https://pytorch.org/docs/stable/fsdp.html

Reply:
The fsdp link doesn't include the apis for the wrapping policies. I suggest that this is skipped in nitpicky and added back once PyTorch fixes their docs to include that module

Reply:
Do you think it was bug in their docs and they are going to fix it?